test/googleAnalyticsItem.spec.js

```diff
@@ -0,0 +1,44 @@
+const expect = require('chai').use(require('chai-subset')).expect,
+    GoogleAnalyticsItem = require('../web/assets/js/GoogleAnalyticsItem');
+
+describe('GoogleAnalyticsItem', () => {
+    let gaItem;
+
+    beforeEach(() => {
+        gaItem = new GoogleAnalyticsItem('event', 'Downloads', 'Download', 'Label', 'Transport');
+    });
+
+    it('should return instance of GoogleAnalyticsItem', () => {
+        expect(gaItem).to.be.instanceOf(GoogleAnalyticsItem);
+    });
+
+    it('should be an object with own property hitType', () => {
```

Comment:
Can't all those test cases, checking if a property exists on an object, be in one test case checking all properties at once?